testing_utilities/solar_system_factory.hpp

```diff
@@ -84,6 +84,10 @@ class SolarSystemFactory : not_constructible {
     AllBodiesAndOblateness,
   };
 
+  template<typename Frame>
```

Comment:
It's odd/tasteless that only this method is parameterized by the Frame.  Shouldn't the entire class be templated?

Reply:
The other methods really should return an ICRFJ2000Equator system, ignore_frame is a bit of a hack that I'd like to keep in one place.